### PR TITLE
Improves Oldstation/Charlie Station

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
tweak: Charlie/Delta Station (the "old cryogenics pod" ghost role) has been improved. Survivors can now finish the singularity engine and will have more minerals available inside the Hivebot mothership. The local atmos network is connected to an air tank; fire alarms and a bathroom have been added. Keep an eye out for a defibrillator too.
/:cl:

Oldstation/Charlie Station was in need of some improvements (wall of text incoming):
* It had no bathroom. Is that why everyone left?
* Atmos had no air tank connected to it.
* Minerals inside the hivebot mothership were very sparse. I increased the amounts and added titanium - bluespace crystals/diamonds are untouched so players still can't build a teleporter without outside help.
* The singularity engine was missing PA parts and plasma tanks; I added both.
* I added a "don't abandon your fellow survivors" line to spawners. This should be extremely obvious, but you still see people grab the spacesuits and fuck off towards the station every now and then.
* Chemicals were stored in open beakers. I switched them out with bottles. This reduces the amount from 50u to 30u, which is no big deal since everybody builds chem dispensers anyway.
* Plasma tank/firing pin crates are locked by default and can be unlocked with the engineer's/sec officer's ID.
* Added a defib to space so players have a chance at reviving dead people.

Misc tweaks:
* Changed power input cable color to yellow to make it easier for new players to distinguish power input from output.
* Transit tube station now open on the northern side (holy shit this was annoying).
* The atmos vent/pump in the broken western section now start turned off since they were throwing runtimes.
* Added fire alarms and a few firelocks - both stations already had a bunch of firelocks, but no alarms.
* Deleted the technology disk since it's no longer used with techwebs. AFAIK there is no way to add the same functionality without also giving the main station all research nodes on upload.
* Made ID cards more descriptive and added a short PACMAN "manual". Replaced most standard crates with appropriate med/engineering subtypes.

### CODE REVIEW

**Comment by 81Denton** [comment]
Does the whole station have a name other than oldstation? If no I'll probably leave the ID titles as "Charlie Station Engineer" etc.

**Comment by Anonmare** [comment]
Can we see some screenshots?

**Comment by 81Denton** [comment]
@Anonmare 
Generator room with colored nodes so new players have a clue how to wire it
old01
Bathroom
old02
New engineering storage with PA parts
old03
Chemical storage
old04
Recolored solar wires
old05

File: _maps/RandomRuins/SpaceRuins/oldstation.dmm [hunk 1]
```diff
@@ -632,12 +639,13 @@
...
 "cf" = (
 /obj/effect/decal/cleanable/dirt,
-/obj/structure/transit_tube/station/reverse/flipped{
-	dir = 1
-	},
 /obj/structure/transit_tube_pod{
 	dir = 4
 	},
+/obj/structure/transit_tube/station/reverse{
+	icon_state = "closed_terminus0";
```
Comment (ShizCalev) [inline _maps/RandomRuins/SpaceRuins/oldstation.dmm]: dv

File: _maps/RandomRuins/SpaceRuins/oldstation.dmm [hunk 2]
```diff
@@ -3541,12 +3563,13 @@
...
 "jq" = (
 /obj/effect/decal/cleanable/dirt,
-/obj/structure/transit_tube/station/reverse/flipped{
-	dir = 1
-	},
 /obj/structure/transit_tube_pod{
 	dir = 4
 	},
+/obj/structure/transit_tube/station/reverse{
+	icon_state = "closed_terminus0";
```
Comment (ShizCalev) [inline _maps/RandomRuins/SpaceRuins/oldstation.dmm]: dv

File: _maps/RandomRuins/SpaceRuins/oldstation.dmm [hunk 3]
```diff
@@ -4160,9 +4185,9 @@
 /turf/open/floor/plating/airless,
 /area/ruin/space/has_grav/ancientstation/atmo)
 "kW" = (
-/obj/machinery/atmospherics/components/binary/pump/on{
-	dir = 8;
-	name = "Air Outlet Pump"
+/obj/machinery/atmospherics/components/binary/pump{
+	icon_state = "pump_map";
```
Comment (ShizCalev) [inline _maps/RandomRuins/SpaceRuins/oldstation.dmm]: dv

File: _maps/RandomRuins/SpaceRuins/oldstation.dmm [hunk 4]
```diff
@@ -4354,41 +4385,357 @@
...
+/area/ruin/space/has_grav/ancientstation)
+"Ql" = (
+/obj/machinery/door/airlock,
+/turf/open/floor/plating,
+/area/ruin/space/has_grav/ancientstation)
+"Qp" = (
+/obj/effect/decal/cleanable/dirt,
+/obj/machinery/door/window/westright,
+/obj/machinery/shower{
+	icon_state = "shower";
```
Comment (ShizCalev) [inline _maps/RandomRuins/SpaceRuins/oldstation.dmm]: dv

File: _maps/RandomRuins/SpaceRuins/oldstation.dmm [hunk 5]
```diff
@@ -4354,41 +4385,357 @@
...
+"Sf" = (
+/obj/effect/decal/cleanable/dirt,
+/obj/structure/particle_accelerator/fuel_chamber,
+/obj/effect/decal/cleanable/dirt,
+/turf/open/floor/plating,
+/area/ruin/space/has_grav/ancientstation/deltacorridor)
+"SJ" = (
+/obj/effect/decal/cleanable/dirt,
+/turf/open/floor/plasteel/whitepurple/side{
+	icon_state = "whitepurple";
```
Comment (ShizCalev) [inline _maps/RandomRuins/SpaceRuins/oldstation.dmm]: dv

File: _maps/RandomRuins/SpaceRuins/oldstation.dmm [hunk 6]
```diff
@@ -4354,41 +4385,357 @@
...
+/obj/structure/cable/yellow{
+	icon_state = "1-2"
+	},
+/turf/open/floor/plating,
+/area/ruin/space/has_grav/ancientstation/engi)
+"Ym" = (
+/obj/effect/decal/cleanable/dirt,
+/obj/machinery/door/window/westleft,
+/obj/machinery/shower{
+	icon_state = "shower";
```
Comment (ShizCalev) [inline _maps/RandomRuins/SpaceRuins/oldstation.dmm]: dv

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Pull request merged**